DOC-5266 Aling data migration tool summaries, articles, and attributes (#202)
datastax/migration-docs: modules/ROOT/pages/deployment-infrastructure.adoc

== Infrastructure requirements

> To deploy {product-proxy} and its companion monitoring stack, you will have to provision infrastructure that meets the following requirements.

Reviewer: ```suggestion
To deploy {product-proxy} and its companion monitoring stack, you must provision infrastructure that meets the following requirements.
```
